### DISCUSSION TITLE
SmokePing

### DISCUSSION THREAD

**Discussion by adamantike** [selfhst/icons#19] [New Icon Request]
Official website: https://oss.oetiker.ch/smokeping/
Repository: https://github.com/oetiker/SmokePing

**Comment by shollyethan** [comment]
I'm struggling a bit to find a high quality image for SmokePing. I'll leave this open for now in case anyone else has access to one.

**Comment by SimpleHomelab** [comment]
+1. But not sure where to find the icon though. Dashboard icons seem to have one but may be it's from an app?

**Comment by barreeeiroo** [comment]
@shollyethan what are the minimum required dimensions for a logo to be eligible for selfhst icons? Asking as I will give it a try to find a good quality one.

**Reply by shollyethan** [reply]
I don't have any real requirements, but wasn't crazy about not having a high-quality SmokePing icon to start with given the stylized text is blurred and doesn't scale well.

With that being said, I went ahead and made some adjustments to the file you had provided and just uploaded it to the collection. Thanks!
